core/src/test/java/org/hibernate/ogm/utils/AvailableGridDialect.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.utils;
+
+
+/**
+ * The testsuite needs some knowledge on all NoSQL dialects.
+ * We mainly need the name of the GridDialect class.
+ *
+ * @author Mark Paluch
+ */
+public enum AvailableGridDialect {
```

Comment:
Do we really need to define this class?

Can't we just use the Dialect class or class name?